sszgen/main.go

```diff
@@ -148,9 +155,10 @@ func parseInput(source string) (map[string]*ast.File, error) {
 			return nil, err
 		}
 		for _, v := range astFiles {
-			if !strings.HasSuffix(v.Name, "_test") {
-				files = v.Files
+			if strings.HasSuffix(v.Name, "_test") || v.Name == "ignore" {
```

Comment:
This change skips files in the `ignore` package for the same reason it skips `_test` files that may be in a testing package. `grpc-gateway` generates files in the same directory as our protobuf .go files, but with the package name `ignore`. Because this source discovery method essentially grabs the last package name it sees in the given directory, fastssz would fail when these files were present (ie it would see the files in the grpc-gateway package and ignore the types we wanted to do ssz generation for).